groovy/inductor/inductor_xpu_backend_e2e.groovy

```diff
@@ -0,0 +1,304 @@
+node(env.nodes_label){
+    cleanWs()
+    stage('setup env') {
+        println('================================================================')
+        println('setup nodes env')
+        println('================================================================')
+        sh '''
+        set -e
+        set +x
+        if [ -e ${HOME}/miniconda3/etc/profile.d/conda.sh ];then
+            source ${HOME}/miniconda3/etc/profile.d/conda.sh 2>&1 >> /dev/null
+            if [ -d ${HOME}/miniconda3/envs/${conda_env} ];then
+                conda activate ${conda_env}
+            else
+                conda create -n ${conda_env} cmake ninja pillow python=${python_version}
+            fi
+        else
+            wget -q -e "https_proxy=http://proxy-us.intel.com:912" https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh -b
+            source ${HOME}/miniconda3/etc/profile.d/conda.sh 2>&1 >> /dev/null
+            conda create -n ${conda_env} cmake ninja pillow python=${python_version}
+        fi
+
+        # set gpu governor
+        if [[ -z "${USER_PASS}" ]];then USER_PASS="gta";fi
```

Comment:
Don't hardcode passwd in this public repo, if we really need such info, let's pass them by Jenkins credential

Reply:
set it as jenkins parameter

Reply:
if the cpu mode setting moved, do we still need this one?